lang/cs/Org.Apache.REEF.Network/Group/Operators/Impl/ScatterReceiver.cs

```diff
@@ -99,5 +96,16 @@ public List<T> Receive()
             _topology.ScatterToChildren(elements, MessageType.Data);
             return elements.ToList();
         }
+
+        /// <summary>
+        /// Ensure all parent and children nodes in the topology are registered with teh Name Service.
+        /// </summary>
+        void IGroupCommOperatorInternal.WaitForRegistration()
```

Comment:
The other implementations don't prefix it with `IGroupCommOperatorInternal.`. Is this needed here?